eth/beacon/types/special_records.py

```diff
@@ -0,0 +1,28 @@
+import rlp
+from rlp.sedes import (
+    binary,
+)
+
+from eth.rlp.sedes import (
+    uint8,
+)
+
+
+class SpecialRecord(rlp.Serializable):
+    """
+    Note: using RLP until we have standardized serialization format.
+    """
+    fields = [
+        # Kind
+        ('kind', uint8),
```

Comment:
Changing this to `uint64` pending https://github.com/ethereum/eth2.0-specs/pull/170
```suggestion
        ('kind', uint64),
```